synapse/handlers/room.py

```diff
@@ -469,13 +469,17 @@ async def clone_existing_room(
             (EventTypes.PowerLevels, ""),
         ]
 
-        # If the old room was a space, copy over the room type and the rooms in
-        # the space.
+        # Copy the room type as per MSC3818 https://github.com/matrix-org/matrix-spec-proposals/pull/3818.
```

Comment:
style nit: Let's avoid overly long lines.
```suggestion
        # Copy the room type as per MSC3818
        # https://github.com/matrix-org/matrix-spec-proposals/pull/3818.
```

Reply:
I think we can just not include the URL -- we usually don't in our code comments.

Reply:
(how do you survive!?!?!?!? I'd go insane if I couldn't click links directly to MSCs)